infrastructure/postgres.tf

```diff
@@ -71,3 +71,18 @@ resource "azurerm_key_vault_secret" "POSTGRES-PASS" {
   key_vault_id = data.azurerm_key_vault.key_vault.id
 }
 
+data "azurerm_client_config" "current" {}
+
+data "azuread_group" "dts_jit_access_juror_db_admin" {
+  display_name = "DTS JIT Access Juror DB Admin"
+}
+
+resource "azurerm_postgresql_flexible_server_active_directory_administrator" "jit" {
+  server_name         = module.postgresql_flexible.server_name
+  resource_group_name = local.rg_name
+  tenant_id           = data.azurerm_client_config.current.tenant_id
+  object_id           = data.azuread_group.dts_jit_access_juror_db_admin.object_id
+  principal_name      = data.azuread_group.dts_jit_access_juror_db_admin.display_name
+  principal_type      = "Group"
```

Comment:
i had similar in https://github.com/hmcts/juror-api/pull/288 the only difference would be `depends_on = [module.postgresql_flexible]` - is this worth adding? the flexible module had the same depends_on so i kept it in